app/templates/build/jira_default.jinja

```diff
@@ -0,0 +1 @@
+*{{ body }}*
```

Comment:
Please add a sample file under docs/sample for this default template, though it's same content with other template like chat_default, it's better for user to understand what data to provide